tests/credentials_tests.rb

```diff
@@ -83,6 +83,7 @@
         aws_secret_access_key: 'dummysecret',
         aws_session_token: 'dummytoken',
         region: 'us-west-1',
+        STS_ENDPOINT: "https://sts.amazonaws.com"
```

Comment:
The all-caps here seems out of place compared to the other keys. Could we use lowercase for consistency?

Reply:
Yep, makes sense to me. I just removed "GLOBAL" from the original variable, I can down case this.